google_guest_agent/non_windows_accounts_integ_test.go

```diff
@@ -1,12 +1,76 @@
+//go:build integration
 // +build integration
 
 package main
 
 import (
+	"fmt"
+	"os"
 	"os/exec"
+	"strings"
 	"testing"
 )
 
+const (
+	testUser       = "integration-test-user"
+	expectedGroups = []string{"adm", "dip", "docker", "lxd", "plugdev", "video", "google-sudoers"}
```

Comment:
this is a configuration value, not a constant. you can set a list here IF you also set it in the config global, otherwise suggest  populating this at runtime with the same code used in `createGoogleUser()`:
```
const (
    defaultgroupstring := "adm,dip,docker,lxd,plugdev,video"
)
[ ... later in the test function ... ]
expectedGroups := config.Section("Accounts").Key("groups").MustString(defaultgroupstring)
```

we may also want a test that explicitly sets this to something else, and validates that it gets used during user creation